bridges/evm/contracts/BridgeV2.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+import {Bridge} from "./Bridge.sol";
+import {IBridgeV2} from "./BridgeV2.sol";
+
+
+contract BridgeV2 is Bridge, IBridgeV2 {
+    struct Request {
```

Comment:
Let's reuse the same request packet struct

Reply:
Noooooooo. It mean use RequestPacket struct in `IBridge.sol`